lib/storage/http.js

```diff
@@ -22,15 +22,22 @@ const DEFAULT_CONTENT_TYPE = 'application/octet-stream';
 
 async function download(source, file) {
     try {
-        const contentType = mime.contentType(source.name)  || DEFAULT_CONTENT_TYPE;
+        const contentType = mime.contentType(source.name) || DEFAULT_CONTENT_TYPE;
         console.log(`downloading source: ${AssetComputeLogUtils.redactUrl(source.url)} into ${file}`);
+
+        const headers = {
```

Comment:
However, I highly **doubt we even have to set the `content-type` header**. After all we are just downloading a binary file.

We guess it based on the source extension name which is fragile anyway, and I don't think Azure Blob Store or S3 require it in any form. How are you supposed to know the content type before you retrieve it?

This might just be a relict from some old code that didn't yet know what to do :) Would be worth removing and testing (and integration testing) while we touch this code.

Reply:
I suggested this change earlier but there was some reason to log the that we were adding custom headers^